### DISCUSSION TITLE
Suddenly dashPanel is showing an incorrect OCP and I'm unable to controll anything

### DISCUSSION THREAD

**Discussion by lockeandev** [tagyoureit/nodejs-poolController#774] [Q&A]
DashPanel says my Outdoor panel is an intellitouchi9+3. But my panel is actually an intellicenter i8ps. And the dashPanel looks odd. 
![Screenshot_20230602_225147_Chrome](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/be0b03cc-e5ce-4963-8143-bb9f57c94f15)

I tried updating to the latest versions of njsPC and dashPanel but no change.

**Comment by rstrouse** [comment]
This seems to indicate some sort of comms issue.  Please create a replay.  You may need to pull dashPanel as of this AM because there was a fix for the popovers.

**Comment by rstrouse** [comment]
Wait a minute did you by chance update the IntelliCenter to 2.006.  If you did then you need to roll it back.  Pentair has since pulled that firmware.

**Comment by lockeandev** [comment]
I don't remember updating it, but it looks like I do indeed have version 2.006. did they enable automatic updates or something? Im out of town for a summer vacation, but here is a screenshot of the the about screen from the pentair intellicenter app:  
![Screenshot_20230604_220846_IntelliCenter2](https://github.com/tagyoureit/nodejs-poolController/assets/11232490/62a582ea-b784-4a98-8d8a-7f77a8b37080)

If that is version 2.006, how do I go about rolling it back and what version do I use?

**Comment by rstrouse** [comment]
Roll back to 1.064. You can download the firmware from the Pentair website.  There have been reports of folks getting an auto update.  But I suspect they tapped the little update icon by accident.

**Reply by tagyoureit** [reply]
I added a note for this to the top of the [Readme.md](https://github.com/tagyoureit/nodejs-poolController) as well.

**Comment by lockeandev** [comment]
Rolling back to 1.064 seems to have fixed my issues.
